supabase/migrations/20230811160337_enable-multiple-apis-per-project.sql

```diff
@@ -0,0 +1,85 @@
+create table "public"."apis" (
+    "id" uuid not null default gen_random_uuid(),
```

Comment:
i don't think this is worth changing but is there a reason not just to use an int?

Reply:
Neither is _better_, only trade-offs. Read more [here](https://www.baeldung.com/uuid-vs-sequential-id-as-primary-key)

I think the most useful property of uuids is they won't collide with ids from other tables (e.g. you couldn't accidentally query for an api with the org id, whereas with integers this can happen)